pkg/types/clusterconfig/clusterconfig.go

```diff
@@ -1018,6 +1026,19 @@ func validateBucketName(bucket string) (string, error) {
 	return bucket, nil
 }
 
+func validateVPCCIDR(cidr string) (string, error) {
+	if cidr == "" {
```

Comment:
Should empty string be allowed? If not, we can remove this check.

Reply:
EDIT: I think empty is probably already blocked by the config reader. So we can probably drop this check